pkg/asset/installconfig/powervs/client.go

```diff
@@ -388,8 +388,16 @@ func (c *Client) GetDNSInstancePermittedNetworks(ctx context.Context, dnsID stri
 }
 
 // CreateDNSRecord Creates a DNS CNAME record in the given base domain and CRN.
-func (c *Client) CreateDNSRecord(ctx context.Context, crnstr string, baseDomain string, hostname string, cname string) error {
-	logrus.Debugf("CreateDNSRecord: crnstr = %s, hostname = %s, cname = %s", crnstr, hostname, cname)
+func (c *Client) CreateDNSRecord(ctx context.Context, publish string, crnstr string, baseDomain string, hostname string, cname string) error {
+	if publish == string(types.InternalPublishingStrategy) {
+		return c.createPrivateDNSRecord(ctx, crnstr, baseDomain, hostname, cname)
+	} else {
+		return c.createPublicDNSRecord(ctx, crnstr, baseDomain, hostname, cname)
+	}
```

Comment:
>  pkg/asset/installconfig/powervs/client.go:394:9: indent-error-flow: if block ends with a return statement, so drop this else and outdent its block (revive)

Maybe turn this into a case statement for `types.InternalPublishingStrategy` `types.ExternalPublishingStrategy` and then end the function with a return of `Error unknown publishing strategy`

Reply:
Thanks, addressed in the latest push.